packages/mdc-textfield/constants.ts

```diff
@@ -34,12 +34,15 @@ const cssClasses = {
   DENSE: 'mdc-text-field--dense',
   DISABLED: 'mdc-text-field--disabled',
   FOCUSED: 'mdc-text-field--focused',
+  FULL_WIDTH: 'mdc-text-field--fullwidth',
   HELPER_LINE: 'mdc-text-field-helper-line',
   INVALID: 'mdc-text-field--invalid',
+  NO_LABLE: 'mdc-text-field--no-label',
```

Comment:
LABEL

Reply:
@moog16
I guess mdc constant follows below rule...

1. remove "mdc-text-field--"
2. apply snakecase
3. apply uppercase

Why LABLE is suitable in this case?

Reply:
I don't think I understand what you're trying to say. I'm saying that LABLE is misspelled.

Reply:
Oh I'm so sorry.
I understood you want to change fron NO_LABEL to LABEL.
I'll change it!